api/firms/firms.go

```diff
@@ -36,7 +40,7 @@ func FirmsTop() func(w http.ResponseWriter, r *http.Request) {
 		defer conn.Close()
 		query := fmt.Sprintf(`
 SELECT id, name, balance, size, execs,
-tax, rank, private, last_payout
```

Comment:
Missing assocs here, I will add them

Reply:
I think this was from when you accidentally undid the assocs API change I added, but it's alright